draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1028,11 +1028,16 @@ POST https://\<log server>/ct/v2/add-chain
 
 Inputs:
 
+: precertificate:
+  : The base64 encoded precertificate.  This field MUST be included if and only
+    if the certificate is being submitted as a precertificate.
+
 : chain:
-  : An array of base64 encoded certificates. The first element is the
-    certificate for which the submitter desires an SCT; the second certifies the
-    first and so on to the last, which either is, or is certified by, an
-    accepted trust anchor.
+  : An array of base64 encoded certificates. If the certificate is being
+    submitted as a precertificate, then the first element is the signer of the
+    precertificate; otherwise, the first element is the certificate being
+    submitted.  The second certifies the first and so on to the last, which
+    either is, or is certified by, an accepted trust anchor.
 
 Outputs:
```

Comment:
To not "break" the document while we move the SCT type into the SCT data structure itself, the outputs section for add-chain should indicate that the type of the returned `TransItem` is either x509_sct_v2 OR precert_sct_v2